### PR TITLE
Checkbox UI

### PR DESCRIPTION
**Pull request by ximex**
removed unused function `form_multi_checkboxes`
style checkboxes with bootstrap classes

before:
grafik
after:
grafik

### CODE REVIEW

File: includes/pages/user_shifts.php [hunk 1]
```diff
@@ -372,10 +372,11 @@ function make_select($items, $selected, $name, $title = null, $additionalButtons
...
 
     foreach ($items as $i) {
-        $htmlItems[] = '<div class="checkbox">'
-            . '<label><input type="checkbox" name="' . $name . '[]" value="' . $i['id'] . '" '
+        $id = $name . '_' . $i['id'];
+        $htmlItems[] = '<div class="form-check">'
+            . '<input class="form-check-input" type="checkbox" id="' . $id . '" name="' . $name . '[]" value="' . $i['id'] . '" '
             . (in_array($i['id'], $selected) ? ' checked="checked"' : '')
-            . ' > ' . $i['name'] . '</label>'
+            . ' /><label class="form-check-label" for="' . $id . '">' . $i['name'] . '</label>'
```
Comment (MyIgel) [inline includes/pages/user_shifts.php:379]: xhtml is not needed hare?

Comment (MyIgel) [reply includes/pages/user_shifts.php:379]:
```suggestion
            . ' ><label class="form-check-label" for="' . $id . '">' . $i['name'] . '</label>'
```

**Comment by ximex** [comment]
I'm a fan of the self-closing tags. also things like `disabled="disabled"`. but could change it if not wanted

**Comment by MyIgel** [comment]
That would be great :)

**Pull request merged**